torch/utils/_sympy/functions.py

```diff
@@ -19,6 +30,8 @@
 from .numbers import int_oo
 
 
+T = TypeVar("T", bound=SupportsFloat)
```

Comment:
Unlikely this TypeVar should be public, should probably begin with `_T` ('in both variable and name)` so it's not exported.